editions/tw5.com/tiddlers/DevOps.tid

```diff
@@ -0,0 +1,8 @@
+created: 20230723073117741
+modified: 20230723073327807
+tags: Definitions
+title: DevOps
```

Comment:
I don't think we need a definition for "DevOps"; the word can just omitted from the GitLab tiddler